client/app/components/blueprints/blueprint-editor/_blueprint-editor.sass

```diff
@@ -1,3 +1,6 @@
+.capitalize
```

Comment:
There exists angular filter uppercase, would recommend use of that vs style class

https://docs.angularjs.org/api/ng/filter/uppercase